tests/parallel_testsuite.py

```diff
@@ -22,6 +23,7 @@
 
 
 def g_testing_thread(work_queue, result_queue, temp_dir):
+  signal.signal(signal.SIGINT, signal.SIG_IGN)
```

Comment:
maybe a comment here? I'm not sure what this does or how it helps.

Reply:
Yeah I don't speak signal